git_utils.py

```diff
@@ -411,6 +415,19 @@ def determinePathToGitRepo( packagePath, verbose = False ):
     # Check under the root of the git repo area for a bare repo w/ the right name
     gitRoot = determineGitRoot()
     gitPackageDir  = packageName + ".git"
+    if verbose:
+        print( "determinePathToGitRepo: gitRoot is %s" % gitRoot if gitRoot else None )
+
+    # Check github DEF_GITHUB_REPOS, just add the package name
+    defRepoPath = DEF_GITHUB_REPOS + '/' + gitPackageDir
+    if 'github.com' in gitRoot:
+        # This results in an extra github API access, but is needed to test if the package is hosted on github
+        repoTags = gitGetRemoteTags( defRepoPath, verbose=verbose )
+        if len(repoTags) > 0:
+            if verbose:
+                print( "determinePathToGitRepo: github %s repo is %s" % (packagePath, defRepoPath) )
+            return defRepoPath
```

Comment:
Another option here might be to check if `$GIT_SITE_TOP/path/to/repo.git/hooks/pre-receive` exits with an error not. The modules & packages that we migrated to GitHub have a pre-receive hook that prints an error message and exits return code 1. If the pre-receive hook exits with 0, then you can do the extra github.com check.